Packs/ZeroFox/Integrations/ZeroFox/ZeroFox.py

```diff
@@ -1,5 +1,7 @@
+from enum import Enum
 import demistomock as demisto
 from CommonServerPython import *
+from requests.exceptions import JSONDecodeError
```

Comment:
```suggestion
```
There’s no need to import JSONDecodeError separately. You can use it directly through the json module, for example: json.JSONDecodeError.

Reply:
thank you! corrected